server/lib/generate-alerts/sample-data/mitre.js

```diff
@@ -26,8 +26,8 @@ export const arrayMitreRules = [
     nist_800_53: ['AU.6', 'AU.14', 'AU.5'],
     tsc: ['CC7.2', 'CC7.3', 'CC6.8'],
     mitre: { tactic: ['Defense Evasion'], id: ['T1089'], technique: ['Disabling Security Tools'] },
-    groups: ['ossec'],
-    description: 'Ossec agent disconnected.',
+    groups: ['wazuh'],
+    description: 'Wazuh agent disconnected.',
```

Comment:
Will the rule description change to use `Wazuh` instead of `Ossec`? Could we check with the team?